<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>matrix</class>
 <widget class="QMainWindow" name="matrix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnCol_min1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCol_plus1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Матрица А</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Матрица В</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>771</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="table_1">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTableWidget" name="table_2">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTableWidget" name="table_3">
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_chisloA">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Умножить А на число</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineChislo">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>410</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>450</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>A + B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeduc">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>450</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>A - B</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Матрица С</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMulti">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>450</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>A x B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMinor">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Минор А</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineRow">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>490</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineCol">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>490</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTrans">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>450</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Транс A </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDet">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>450</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>|A|</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineDet">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>490</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Колонки</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Строки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRow_min1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>350</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRow_plus1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>350</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRow_min2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>350</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRow_plus2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>350</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCol_min2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>380</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCol_plus2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>380</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMinorDet">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>|A| минора</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineMinDet">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>570</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="rowSpinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>530</y>
      <width>42</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="colSpinBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>530</y>
      <width>42</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_inv">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>600</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Обратная A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAlgDop">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>600</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Матрица алг доп</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAlg">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>530</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Алг доп</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="rowSpinBox_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>530</y>
      <width>42</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="colSpinBox_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>530</y>
      <width>42</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineAlgDop">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>570</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSolve">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>600</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>AX=B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_minorElem">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>600</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>М для элем</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>40</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>История вычислений</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>829</x>
      <y>69</y>
      <width>311</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout"/>
   </widget>
   <widget class="QPushButton" name="btn_chistka">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>660</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMoveA">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>660</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Переместить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineMove">
    <property name="geometry">
     <rect>
      <x>962</x>
      <y>660</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1176</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMatrix">
    <property name="title">
     <string>Matrix</string>
    </property>
   </widget>
   <addaction name="menuMatrix"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
